src/components/Svg.tsx

```diff
@@ -1,46 +1,47 @@
 ///<reference path="../../node_modules/@types/webpack-env/index.d.ts"/>
 import * as React from 'react';
 
-let svgsEnum = require('../../node_modules/coveo-styleguide/dist/svg/CoveoStyleGuideSvg.json') as { [key: string]: string };
-
-const parseSVG = (str: string): SVGElement => {
-  let container = document.createElement('div');
-
-  container.innerHTML = str;
-
-  return container.children[0] as SVGElement;
-};
+import '../utils/Polyfills';
 
-const getElementAttributes = (element: Element): { [key: string]: string } => {
-  let attibutes: { [key: string]: string } = {};
+let svgsEnum = require('../../node_modules/coveo-styleguide/dist/svg/CoveoStyleGuideSvg.json') as { [key: string]: string };
 
-  for (let i = 0; i < element.attributes.length; i++) {
-    let attr: Attr = element.attributes[i];
-    attibutes[attr.name] = attr.value;
-  }
+const setSvgClass = (svgString: string, svgClass: string): string => {
+  let parser = document.createElement('div');
+  parser.innerHTML = svgString;
 
-  return attibutes;
+  let svgElement: SVGElement = parser.children[0] as SVGElement;
+  svgElement.setAttribute('class', svgClass);
+  return svgElement.outerHTML; // Polyfill required here!
```

Comment:
Why do you put the polyfill in another file? I think it would be better to have the fix directly here